src/components/Typography/Typography.tsx

```diff
@@ -0,0 +1,69 @@
+import { Text, StyleSheet } from "react-native";
+
+const styles = StyleSheet.create({
+  h1: {
+    display: "flex", //only accepts none or flex
+    fontSize: 2,
+    marginTop: 0.67,
+    marginBottom: 0.67,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+  h2: {
+    display: "flex",
+    fontSize: 1.5,
+    marginTop: 0.83,
+    marginBottom: 0.83,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+  h3: {
+    display: "flex",
+    fontSize: 1.17,
+    marginTop: 1,
+    marginBottom: 1,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+  h4: {
+    display: "flex",
+    fontSize: 1,
+    marginTop: 1.33,
+    marginBottom: 1.33,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+  h5: {
+    display: "flex",
+    fontSize: 0.83,
+    marginTop: 1.67,
+    marginBottom: 1.67,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+  h6: {
+    display: "flex",
+    fontSize: 0.67,
+    marginTop: 2.33,
+    marginBottom: 2.33,
+    marginLeft: 0,
+    marginRight: 0,
+    fontWeight: "bold",
+  },
+});
+
+type Props = {
+  variant: "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
I've been thinking about this and I don't like that this forces a web idea (h1, h2, etc) on RN, but on the other hand "heading" is a valid concept and having headings of different levels makes sense so why not "h1"? 

Point being... if you come up with better names for these variants at some point we should talk about it.

Reply:
I'm gonna review the article a bit and see if there's an approach we can go with in terms of naming. 

I'll consult my gut as well. May get the idea while working on the other components.